### PR TITLE
[FIXES]: Sorted by trend, edit/delete post

### PR DESCRIPTION
**Pull request by shivanigangadharan**
- Implemented Edit/Delete post functionality
- Sort by latest
- Sort by trending
- Fix to post either text/picture or both

### CODE REVIEW

File: src/pages/homepage/homepage.js [hunk 1]
```diff
@@ -84,7 +94,7 @@ export default function Homepage() {
                 <div className="create-post-container">
                     <img alt="profile-pic" src={user.profilepic} className="avatar" />
                     <div className="create-post-content">
-                        <textarea onChange={(e) => { setPostContent(e.target.value); }} placeholder="Write something fun here, to post..." className="create-post-input"></textarea>
+                        <textarea value={postContent} onChange={(e) => { setPostContent(e.target.value); }} placeholder="Write something fun here, to post..." className="create-post-input"></textarea>
```
Comment (sharathnair9999) [inline src/pages/homepage/homepage.js]: If the onChange is having only one function then the  curly brace around it is not needed

File: src/pages/explore-page/explore-page.js [hunk 2]
```diff
@@ -12,6 +12,15 @@ export default function ExplorePage() {
...
     const [users, setUsers] = useState([]);
+
+    const sortByTrending = () => {
+        setPosts((post) => [...post.sort((a, b) => { return b.likes - a.likes })]);
+    }
+
+    const sortByLatest = () => {
+        setPosts((post) => [...post.sort((a, b) => { return b.createdAt - a.createdAt })]);
+    }
+
```
Comment (abhishektanwar) [inline src/pages/explore-page/explore-page.js:23]: suggestion : Functions like these could be moved to a helper.js file

**Review by abhishektanwar — APPROVED** [review]
I have made a suggestion, please have a look at that.

Code is clean and understandable. Good to merge 
Good work shivani

**Review by sharathnair9999 — APPROVED** [review]

**Review by spraveenofficial — APPROVED** [review]
Looks good. You can also handle image upload error, if uploading from client side.

File: src/pages/homepage/homepage.js (hunk 1 above)
Comment (shivanigangadharan) [reply src/pages/homepage/homepage.js]: done

**Pull request merged**